### PR TITLE
Additions and fixes for lesson feature

### LINKED ISSUE #435: Additions and fixes for lesson feature
- Moderator/Teacher should be able to activate/deactivate learning modules, lessons.
- Fix mentioned in issue #439

### PR DESCRIPTION
**Pull request by adityacp**
This PR fixes issues mentioned in #435 
Closes #435

### CODE REVIEW

File: yaksh/test_models.py [hunk 1]
```diff
@@ -204,6 +207,29 @@ def test_get_status(self):
         # Then
         self.assertEqual(status, module_status)
 
+        # Module in progress
+
+        # Given
+        self.course_status.completed_units.add(self.learning_unit_one)
```
Comment (maheshgudi) [inline yaksh/test_models.py]: The test case name is a bit confusing.` test_get_course_status` is more apt, IMHO.

Comment (maheshgudi) [inline yaksh/test_models.py]: Making a change on a object created during setup can cause a lot of problems in further testcases. Please avoid to add/delete/update class objects in a class method or atleast revert them back at the end of the test case.

File: yaksh/test_models.py [hunk 2]
```diff
@@ -204,6 +207,29 @@ def test_get_status(self):
...
+        # When
+        status = self.learning_module.get_status(self.student, self.course)
+        # Then
+        self.assertEqual("inprogress", status)
+
+        # Module is completed
+
+        # Given
+        self.course_status.completed_units.add(self.learning_unit_two)
```
Comment (maheshgudi) [inline yaksh/test_models.py]: same here

File: yaksh/views.py [hunk 3]
```diff
@@ -2348,6 +2353,14 @@ def show_lesson(request, lesson_id, module_id, course_id):
     learn_module = course.learning_module.get(id=module_id)
     learn_unit = learn_module.learning_unit.get(lesson_id=lesson_id)
     learning_units = learn_module.get_learning_units()
+
+    # if learning module is active
```
Comment (prathamesh920) [inline yaksh/views.py]: if learning module is not active.
You an omit the comment if you want because code indicates the intention here.

**Comment by prathamesh920** [comment]
LGTM

File: yaksh/test_models.py (hunk 2 above)
Comment (adityacp) [reply yaksh/test_models.py]: resolved

File: yaksh/test_models.py (hunk 1 above)
Comment (adityacp) [reply yaksh/test_models.py]: resolved

Comment (adityacp) [reply yaksh/test_models.py]: changed the name to `test_get_module_status` since module status is being tested.

**Pull request merged**